cmd/puppet-db/export.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var exportViper = viper.New()
+
+var exportCmd = &cobra.Command{
+	Use:   "export",
+	Short: "export <path> [--anon=<profile>]",
+	Long:  "export <path> [--anon=<profile>]",
```

Comment:
I don't think we need to define the `Long` field at all, as I think by default will show the `Short` value